### PR TITLE
feat(ui): improve usability of "reference attribute" form

### PR DESCRIPTION
**Pull request by martinmaillard**
Present the form as a sentence to make it (maybe) easier to understand.

This is the result of what was discussed with @ktk and @l00mi today.

It will probably cause conflicts with #166. Will fix after one of them is merged.

### CODE REVIEW

File: ui/src/components/project/ReferenceAttributeForm.vue [hunk 1]
```diff
@@ -3,44 +3,45 @@
...
+        <b-input type="text" id="property" v-model="attribute.predicateId" required />
       </b-field>
 
-      <b-field label="Identifier columns mapping" v-if="referencedTable" :addons="false">
-        <p class="help">
-          The identifier for <TableTag :table="referencedTable" /> is defined as <code>{{ referencedTable.identifierTemplate }}</code><br>
-          Which columns should be used to reference it?
+      <div v-if="referencedTable">
+        <p>
+          The identifier <code>{{ referencedTable.identifierTemplate }}</code> shall use
```
Comment (tpluscode) [inline ui/src/components/project/ReferenceAttributeForm.vue]: How about "map columns" and not "use columns"?

File: ui/src/components/project/ReferenceAttributeForm.vue [hunk 2]
```diff
@@ -3,44 +3,45 @@
...
-    <section class="modal-card-body">
-      <b-field label="Table">
-        <b-select v-model="attribute.referencedTableId">
-          <option v-for="table in tables" :key="table.id" :value="table.id">{{ table.name }}</option>
+    <section class="modal-card-body reference-form">
+      <p>
+        On table <TableTag :table="table" class="is-medium" />,
+      </p>
+      <p>
+        create a link to the <label for="referenced-table"><strong>category table</strong></label>
```
Comment (tpluscode) [inline ui/src/components/project/ReferenceAttributeForm.vue]: Hm, so we're renaming the entire `Dimension table` concept with the term `Category table`?

Comment (martinmaillard) [reply ui/src/components/project/ReferenceAttributeForm.vue]: Yeah we decided on "Dimension" -> "Category" and "Fact" -> "Measurements". Also "Project" -> "Cube". Will do the renaming in a separate PR.

File: ui/src/components/project/ReferenceAttributeForm.vue (hunk 1 above)
Comment (martinmaillard) [reply ui/src/components/project/ReferenceAttributeForm.vue]: What do you think @ktk @l00mi ?

File: ui/src/components/project/ReferenceAttributeForm.vue (hunk 2 above)
Comment (ktk) [reply ui/src/components/project/ReferenceAttributeForm.vue]: I mentioned this in chat, this is still aligned with Wikipedia entries of cubes but it's closer to the target audience. API will stay the way it is but this is for the users itself.

File: ui/src/components/project/ReferenceAttributeForm.vue (hunk 1 above)
Comment (ktk) [reply ui/src/components/project/ReferenceAttributeForm.vue]: Both are fine for me. But I would not waste too much time on that right now, this is something for next phase.

File: ui/src/components/project/ReferenceAttributeForm.vue (hunk 2 above)
Comment (tpluscode) [reply ui/src/components/project/ReferenceAttributeForm.vue]: > API will stay the way it is but this is for the users itself.

this separation can easily lead to errors and confusion. at some point we should unify the terminology across the board (UI, API, code, internal vocabulary)

**Pull request merged**